src/ansys_sphinx_theme/announcements.html

```diff
@@ -0,0 +1 @@
+<p>You are not reading the most recent version of this documentation. Latest stable release is <a href="https://actions.pyansys.com/release/1.0">1.0</a></p>
```

Comment:
This file is not required. It can only live in the `gh-pages` of the repository.

Reply:
Sure, this i added just for testing and we should remove it before merging.